Configuration/TypoScript/constants.txt

```diff
@@ -50,5 +50,11 @@ plugin.tx_dpf {
 
 			# cat=plugin.tx_dpf/link; type=int+; label=PID of the landing page (Backend Preview)
 			previewPid =
+
+		    # cat=plugin.tx_dpf/link; type=int+; label=Workspace items per page (Backoffice)
+            workspaceItemsPerPage = 30
+
+		    # cat=plugin.tx_dpf/link; type=int+; label=University collection value (Backoffice)
+            universityCollection = epflicht
```

Comment:
What is the function of "universityCollection"?

Reply:
If a publiakation belongs to the configuered unviversity collection it has been emerged from scientific work at the university. In our case this is the "UL". See also Ticket CMR-685.